src/Pact/Analyze/Check.hs

```diff
@@ -651,13 +656,18 @@ verifyCheck
   -> ExceptT VerificationFailure IO CheckResult
 verifyCheck moduleData funName check = do
   let info       = dummyInfo
-      moduleName = moduleData ^. _1.mName
+      module'    = moduleData ^. mdModule
+      moduleName = nameOf module' 
       modules    = HM.fromList [(moduleName, moduleData)]
       moduleFun :: ModuleData -> Text -> Maybe Ref
-      moduleFun (_mod, modRefs) name = name `HM.lookup` modRefs
+      moduleFun ModuleData{..} name = name `HM.lookup` _mdRefMap
 
   tables <- withExceptT ModuleParseFailure $ moduleTables modules moduleData
   case moduleFun moduleData funName of
     Just funRef -> ExceptT $
       Right . head <$> verifyFunctionProps tables funRef [Located info check]
     Nothing -> pure $ Left $ CheckFailure info $ NotAFunction funName
+  where
+    nameOf m = case m of
+      Interface{..} -> _interfaceName
+      Module{..}    -> _mName
```

Comment:
This is a candidate for why we need to onboard prisms into the Pact base - @slpopejoy. `Term` has a prism structure we could take advantage of.